src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        this.color = color;

```

Comment:
This could be confusing for a developer to see such a statement and no color variable defined in this class, so to avoid confusion it is better to call super class constructor super(color);